recipes-support/swupdate/swupdate_%.bbappend

```diff
@@ -1,8 +1,4 @@
 # Configure swupdate client/agent for our purposes.
 # In general we only want the minimum functionality
 # required to verify and install an image file.
-FILESEXTRAPATHS_append := "${THISDIR}/${PN}:"
-
-PACKAGECONFIG_CONFARGS = ""
```

Comment:
Is the deletion of this necessary? I'd like to keep the repo as close as possible to the original files provided by Azure, so I guess this commit would almost be best as a reversion of https://github.com/munit-solutions/meta-azure-device-update/commit/81d1ab6bca5be3dc12d4f77e7ee4d566a7639a4a, right?